pages/_app.tsx

```diff
@@ -1,15 +1,23 @@
-import '../styles/globals.css';
-import type { AppProps } from 'next/app';
-import { Provider } from "react-redux";
-import { store } from "../redux/store";
-import { ThemeProvider } from '@space-metaverse-ag/space-ui';
-
-function App({ Component, pageProps }: AppProps) {
-  return <Provider store={store}>
+import { Provider } from 'react-redux'
+
+import { ThemeProvider, GlobalStyles } from '@space-metaverse-ag/space-ui'
+import type { AppProps } from 'next/app'
+
+import { TopNav, Container } from '~/layouts'
+import { store } from '~/redux/store'
```

Comment:
`~/` style imports are showing errors in my IDE, even though it does seem to load.
Can we either use `.tsconfig` aliases or full relative paths instead of `~/`, this is non-standard.

![image](https://user-images.githubusercontent.com/27584221/198408512-fca24632-39d2-4c02-934b-20b804d9f221.png)